### PR TITLE
add new Kodi Nexus version to test builds

### PR DESCRIPTION
**Pull request by AlwinEsch**
This is a mandatory change to allow builds of binary addons relates to new Kodi Nexus version, without does Jenkins fail!

Please review asap and check/test it, without can no binary addon be updated as it fallbacks to Kodi Leia!

### CODE REVIEW

**Review by Rechi — CHANGES_REQUESTED** [review]
Have you tested this?
There is a syntax error.

**Comment by AlwinEsch** [comment]
Have currently no test system available, thats why I mean above with "check/test it".

Think to see the syntax error, missing comma in json text, sorry!

File: vars/buildPlugin.groovy [hunk 1]
```diff
@@ -9,6 +9,7 @@ def call(Map addonParams = [:])
 	def VERSIONS_VALID = [
 		'Leia': 'leia',
 		'Matrix': 'matrix',
+		'Nexus': 'nexus'
```
Comment (Rechi) [inline vars/buildPlugin.groovy:12]: Why no comma?

File: vars/buildPlugin.groovy [hunk 2]
```diff
@@ -63,7 +64,9 @@ def call(Map addonParams = [:])
 	]
 	def PPA_VERSION_MAP = [
 		'Matrix': [
-			'stable',
+			'stable'
```
Comment (Rechi) [inline vars/buildPlugin.groovy:67]: Why change this line?

File: vars/buildPlugin.groovy (hunk 1 above)
Comment (AlwinEsch) [reply vars/buildPlugin.groovy:12]: The last in list, like on value `def PLATFORMS_VALID = [` below, thought to leave is also not 100% correct (maybe there json works with).

File: vars/buildPlugin.groovy (hunk 2 above)
Comment (AlwinEsch) [reply vars/buildPlugin.groovy:67]: Also to have last value in list without a comma.

File: vars/buildPlugin.groovy (hunk 1 above)
Comment (Rechi) [reply vars/buildPlugin.groovy:12]: This is not JSON, it's groovy.

File: vars/buildPlugin.groovy (hunk 2 above)
Comment (Rechi) [reply vars/buildPlugin.groovy:67]: So that the line has to be changed again, if another item is added to the Matrix list?

**Comment by AlwinEsch** [comment]
In https://github.com/xbmc/pipeline-library/pull/23/files#diff-db2794773b738dbb5566313f115cdf3ff1f45e13e57cbc37aa1bd181819ed5fbR51 is it also with it.
```json
	def UBUNTU_DISTS = [
		'stable': [
			'groovy',
			'focal',
			'bionic',
		],
```

What for you the best to have it always on end or to have removed there too?

File: vars/buildPlugin.groovy (hunk 1 above)
Comment (AlwinEsch) [reply vars/buildPlugin.groovy]: I change it, know thats done by groovy, but looks a bit in json format style to me.

Is updated.

**Comment by wsnipex** [comment]
please add hirsute to Ubuntu stable series.
The rest looks good

**Comment by AlwinEsch** [comment]
> please add hirsute to Ubuntu stable series.
> The rest looks good

Becomes added, thanks!

Makes it also seens to add on nightly the "impish"?

**Comment by wsnipex** [comment]
no, we usually only build for release series, not for betas

**Comment by AlwinEsch** [comment]
Thanks @wsnipex, the Hirsute only is added to stable series now.

**Comment by AlwinEsch** [comment]
Was tests OK about and possible to bring in?

**Review by wsnipex — APPROVED** [review]

**Comment by AlwinEsch** [comment]
@Rechi if OK you, can it come in?

**Pull request merged**